assets/aml-benchmark/components/batch_benchmark_inference_claude/spec.yaml

```diff
@@ -0,0 +1,212 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+type: pipeline
+
+name: batch_benchmark_inference_claude
+display_name: Batch Benchmark Inference with claude support
+description: Components for batch endpoint inference
+version: 0.0.1
+
+inputs:
+  input_dataset:
+    type: uri_folder
+    description: Input jsonl dataset that contains prompt.  For the performance test, this one will be neglected.
+    optional: True
+  model_type:
+    type: string
+    description: Type of model. Can be one of ('aoai', 'oss', 'vision_oss, 'claude')
+    optional: True
+  batch_input_pattern:
+    type: string
+    description: >- 
+      The string for the batch input pattern. The input should be the payload format with substitution
+      for the key for the value put in the `###<key>`. For example, one can use the following format for
+      a llama text-gen model with a input dataset has `prompt` for the payload
+      and `_batch_request_metadata` storing the corresponding ground truth.
+      {
+        "input_data":
+        {
+          "input_string": ["###<prompt>"],
+          "parameters":
+          {
+            "temperature": 0.6,
+            "max_new_tokens": 100,
+            "do_sample": true
+          }
+        },
+        "_batch_request_metadata": ###<_batch_request_metadata>
+      }
+
+      For AOAI model, the following pattern can be used,
```

Comment:
Do we have pattern examples of vision_oss and claude as well?

Reply:
Added patterns for Vision OSS and for Claude, however I used vision OSS example from the unit test.